sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseEditAuthorization.java

```diff
@@ -27,20 +28,19 @@
 	@EJB
 	private CaseService caseService;
 
-	public Boolean caseEditAllowedCheck(String caseUuid) {
+	public Boolean caseEditAllowedCheck(Case caze) {
 
-		Case caze = caseService.getByUuid(caseUuid);	
 		User user = userService.getCurrentUser();
        
-        if (user.getUuid().equals(caseUuid)) {
+        if (DataHelper.equal(user.getUuid(), (caze.getUuid()))) {
```

Comment:
Please use DataHelper.equal for all the checke - especially community and health facility can be null

Reply:
changed